core/api/src/services/pinecone/index.ts

```diff
@@ -0,0 +1,31 @@
+import { Pinecone } from "@pinecone-database/pinecone"
```

Comment:
I would nest this under the openai service. I think treating the 'assistant' service like a domain service (that wraps multiple sub-components) is valid here as the interactions are more complex than a 1-to-1 mapping between some external service calls and an internal representation.